test/integration/package_int_test.go

```diff
@@ -701,6 +701,86 @@ func (suite *PackageIntegrationTestSuite) TestProjectWithOfflineInstallerAndDock
 	cp.ExpectExitCode(0)
 }
 
+func (suite *PackageIntegrationTestSuite) TestImportWithCustomVersionRequirements() {
+	suite.OnlyRunForTags(tagsuite.Package)
+	if runtime.GOOS == "windows" {
+		suite.T().Skip("Skipping windows as builds can be slow and we only really need to test this one one platform")
+		return
+	}
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	// Because the ActiveState-CLI org has enterprise features enabled, we need to login
+	ts.LoginAsPersistentUser()
+
+	complexReqsData := `coverage!=3.5
+docopt>=0.6.1
+Mopidy-Dirble>=1.1,<2
+requests>=2.2,<2.31.0
+urllib3>=1.21.1,<=1.26.5
+`
+
+	cp := ts.Spawn("checkout", "ActiveState-CLI/Comparators-Import", ".")
+	cp.Expect("Skipping runtime setup")
+	cp.Expect("Checked out project")
+	cp.ExpectExitCode(0)
+
+	ts.PrepareFile(filepath.Join(cp.WorkDirectory(), reqsFileName), complexReqsData)
+
+	cp = ts.Spawn("import", reqsFileName)
+	cp.ExpectExitCode(0)
+
+	// Test version numbers once the following story is resolved:
+	// https://activestatef.atlassian.net/browse/DX-2272
+	cp = ts.Spawn("packages")
+	cp.Expect("Mopidy-Dirble")
+	cp.Expect("coverage")
+	cp.Expect("docopt")
+	cp.Expect("requests")
+	cp.Expect("urllib3")
+	cp.ExpectExitCode(0)
+}
+
+func (suite *PackageIntegrationTestSuite) TestProjectWithCustomVersionRequirements() {
+	suite.OnlyRunForTags(tagsuite.Package)
+	if runtime.GOOS == "windows" {
+		suite.T().Skip("Skipping windows as builds can be slow and we only really need to test this one one platform")
+		return
+	}
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	// Because the ActiveState-CLI org has enterprise features enabled, we need to login
+	ts.LoginAsPersistentUser()
+
+	// This project was created with the following requirements.txt:
+	//   coverage!=3.5
+	//   docopt>=0.6.1
+	//   Mopidy-Dirble>=1.1,<2
+	//   requests>=2.2,<2.31.0
+	//   urllib3>=1.21.1,<=1.26.5
+	cp := ts.Spawn("checkout", "ActiveState-CLI/Comparators", ".")
+	cp.Expect("Skipping runtime setup")
+	cp.Expect("Checked out project")
+	cp.ExpectExitCode(0)
+
+	// Uninstall all of the packages with custom version requirements
+	cp = ts.Spawn("uninstall", "coverage")
+	cp.ExpectExitCode(0)
+
+	cp = ts.Spawn("uninstall", "docopt")
+	cp.ExpectExitCode(0)
+
+	cp = ts.Spawn("uninstall", "Mopidy-Dirble")
+	cp.ExpectExitCode(0)
+
+	cp = ts.Spawn("uninstall", "requests")
+	cp.ExpectExitCode(0)
+
+	cp = ts.Spawn("uninstall", "urllib3")
+	cp.ExpectExitCode(0)
+}
```

Comment:
With your latest explanation I think I am caught up. This test should not exist. We are testing an API now, rather than the state tool. That's not our job, we have a hard enough time keeping our tests under control with just covering the State Tool itself.

The new test you added has value but does seem unrelated to the story at hand. Perhaps we should abandon the ticket and file a new one for the `import` test.